### ISSUE TITLE
Crash when applying screenshot plugin with Android Gradle Plugin 3.6 beta 04

### ISSUE THREAD

**Issue by lwasyl** [facebook/screenshot-tests-for-android#233]
I know it's not released yet, but in beta the APIs are probably stable. I get the following crash when configuring the project (btw opened #232 since this should crash on running the task):

```
Caused by: java.lang.NoSuchMethodError: com.android.build.gradle.tasks.PackageAndroidArtifact.getOutputDirectory()Ljava/io/File;
        at com.facebook.testing.screenshot.build.PullScreenshotsTask.init(PullScreenshotsTask.kt:49)
        at com.facebook.testing.screenshot.build.ScreenshotsPlugin.createTask(ScreenshotsPlugin.kt:74)
        at com.facebook.testing.screenshot.build.ScreenshotsPlugin.access$createTask(ScreenshotsPlugin.kt:41)
        at com.facebook.testing.screenshot.build.ScreenshotsPlugin$generateTasksFor$1.execute(ScreenshotsPlugin.kt:85)
        at com.facebook.testing.screenshot.build.ScreenshotsPlugin$generateTasksFor$1.execute(ScreenshotsPlugin.kt:41)
        at org.gradle.api.internal.DefaultCollectionCallbackActionDecorator$BuildOperationEmittingAction$1$1.run(DefaultCollectionCallbackActionDecorator.java:100)
```

Seems like some AGP APIs have changed and need adjustments

**Comment by lawrencebautista** [comment]
Please fix! 3.6 is rc03 now, and this is still occurring.

**Comment by pawegio** [comment]
Any update? It's a blocker for both Gradle 6.2 and AGP 3.6.
